test/opsem/lit.cfg

```diff
@@ -61,6 +68,10 @@ seabmc = [seahorn_cmd, '--horn-bmc-engine=mono',
 seasmt = [seahorn_cmd, '--horn-bmc-engine=mono',   
           '--horn-bmc', '--horn-bv2=true', '--log=opsem',
           '--keep-shadows=true' ]
+sea = [sea_cmd, 'bpf', '--horn-bmc-engine=mono',
```

Comment:
we should start using yama command in tests and store configuration file inside test directory